roles/openshift_metrics/vars/main.yaml

```diff
@@ -18,3 +18,4 @@ hawkular_persistence: "{% if openshift.hosted.metrics.storage.kind != None %}tru
 hawkular_type: "{{ 'origin' if deployment_type == 'origin' else 'enterprise' }}"
 
 metrics_upgrade: openshift.hosted.metrics.upgrade | default(False)
+openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
```

Comment:
I wonder if this is because I moved this definition to `openshift_master_facts` from `openshift_master`?